pytripgui/controller/settings_cont.py

```diff
@@ -0,0 +1,100 @@
+import os
+import pickle
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SettingsController:
+    """
+    Class for interacting with saved configurations.
+    """
+
+    def __init__(self, model):
+        """
+        """
+
+        self.model = model
+        # new settings handler here
+
+        self.pyc_path = os.path.join(self.get_user_directory(), "settings.pyc")
+        logger.debug("New setup config parser based on {:s}".format(self.pyc_path))
+
+        self.load()
+
+    def load(self):
+        """
+        Load confif, sets self.model accordingly.
+        If file does not exist, exit silently.
+
+        It is connected to Model, in a way that
+        - upon SettingsController.load(), SettingsModel attributes starting with "_"  are not written to         Model.
+        - upon SettingsController.save(),         Model attributes starting with "__" are not written to SettingsModel.
+
+        This way,
+            a) _version is written to disk, but imported into Model when loading
+            b) __internal_attribute__ are not passed between Model and SettingsModel
+        """
+
+        logger.debug("SettingsController.load() : loading settings from {}".format(self.pyc_path))
+
+        model = self.model
+        pyc = self.pyc_path
```

Comment:
Why the variable is called `pyc` ? It resembles python compiled file.
Pickle files should have extension `.pkl` or something similar.

Reply:
no idea where i got that from. Will fix.

Reply:
https://stackoverflow.com/questions/40433474/preferred-or-most-common-file-extension-for-pickle-files